lib/ibmdb.js

```diff
@@ -755,49 +845,47 @@ IBMDB.prototype.buildIndex = function(model, property) {
     return '';
   }
 
-  var stmt = 'CREATE ';
-  var kind = '';
+  var statement = new ParameterizedSQL('CREATE');
   if (i.kind) {
-    kind = i.kind;
+    statement.merge(i.kind);
+  } else if (i.unique && i.unique === true) {
```

Comment:
If this value is only ever going to be true, false and null/undefined, then it can just say 
```
else if (i.unique) {
//etc
```